textstat/textstat.py

```diff
@@ -340,12 +347,15 @@ def text_standard(self, text, float_output=None):
         d = Counter(grade)
         final_grade = d.most_common(1)
         score = final_grade[0][0]
-        
+
         if float_output:
             return float(score)
         else:
-            return "{}th and {}th grade".format(
-                str(int(score)-1), str(int(score))
+            lower_score = int(score) - 1
+            upper_score = lower_score + 1
+            return "{}{} and {}{} grade".format(
+                str(lower_score), get_grade_suffix(lower_score),
+                str(upper_score), get_grade_suffix(upper_score)
```

Comment:
`str` isn't needed for `.format()`
```suggestion
                upper_score, get_grade_suffix(upper_score)
```